scss/_buttons.scss

```diff
@@ -131,7 +131,7 @@
   color: #900;
 
   &:hover {
-    color: #fff;
+    color: $text-white;
     background-color: #b33630;
```

Comment:
Okay I'll stop writing "should this be a variable" but looks like there's a bunch of colors in here that aren't variables that perhaps should be.